### ISSUE TITLE
`SQL Azure` incorrectly detected as `SQL Server 2014`

### ISSUE THREAD

**Issue by kgeorge314** [flyway/flyway#2486]
###### Which version and edition of Flyway are you using?
Flyway Community Edition 6.0.0 by Boxfuse

###### If this is not the latest version, can you reproduce the issue with the latest one as well?
(Many bugs are fixed in newer releases and upgrading will often resolve the issue)

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
command-line (Powershell, pswh, cmd)

###### Which database are you using (type & version)?
SQL Azure

###### Which operating system are you using?
Windows 10 64 bit

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)
`.\flyway -user=REDACTED -password='myPassword' -url="jdbc:sqlserver://RedactedRedacted.database.windows.net;databaseName=RedactedRedacted;" -connectRetries=60 migrate`

_corrected typo in URL_

###### What did you expect to see?
Flyway Deployment to Azure SQL Azure

###### What did you see instead?
```
Flyway Community Edition 6.0.0 by Boxfuse
Database: jdbc:sqlserver://REDACTEDREDACTED.database.windows.net:1433;useBulkCopyForBatchInsert=false;cancelQueryTimeout=-1;sslProtocol=TLS;jaasConfigurationName=SQLJDBCDriver;statementPoolingCacheSize=0;serverPreparedStatementDiscardThreshold=10;enablePrepareOnFirstPreparedStatementCall=false;fips=false;socketTimeout=0;authentication=NotSpecified;authenticationScheme=nativeAuthentication;xopenStates=false;sendTimeAsDatetime=true;trustStoreType=JKS;trustServerCertificate=false;TransparentNetworkIPResolution=true;serverNameAsACE=false;sendStringParametersAsUnicode=true;selectMethod=direct;responseBuffering=adaptive;queryTimeout=-1;packetSize=8000;multiSubnetFailover=false;loginTimeout=15;lockTimeout=-1;lastUpdateCount=true;encrypt=false;disableStatementPooling=true;databaseName=REDACTEDREDACTER;columnEncryptionSetting=Disabled;applicationName=Microsoft JDBC Driver for SQL Server;applicationIntent=readwrite; (Microsoft SQL Server 12.0)
ERROR: Flyway Enterprise Edition or SQL Server upgrade required: SQL Server 2014 is no longer supported by Flyway Community Edition, but still supported by Flyway Enterprise Edition.
```

**Comment by kgeorge314** [comment]
Also noticed this error, please let me know if another issue should be created. 
`'eName' is not recognized as an internal or external command,
operable program or batch file.`

ScreenCast of issue. 
![Flyway-6 0 0 0-command-line](https://user-images.githubusercontent.com/7089603/63780269-be734b80-c8df-11e9-8bbd-8dcea723da7f.gif)

**Comment by juliahayward** [comment]
Thanks for the report - the second part (the `eName` issue) has already been reported at https://github.com/flyway/flyway/issues/2487 and we'll answer that one there.

**Comment by juliahayward** [comment]
We have a fix - will be shipped in 6.0.1 and pushed to the repo asap.
